tests/gnrc_rpl_p2p/Makefile

```diff
@@ -0,0 +1,9 @@
+include ../Makefile.tests_common
+
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
```

Comment:
For `native` the application tries to access a TAP interface with `gnrc_netdev_default`. As this is just a compile test you can remove these two modules.